doc/util/versionButton.php

```diff
@@ -89,13 +89,14 @@ function toggleDropDown() {
 function dropSetup() {
   var currentRelease = "1.17";
   var nextRelease = "1.18";
+  var nextNextRelease = "1.19";
   var button = document.getElementById("versionButton");
   // Uses unicode down-pointing triangle
   var arrow = " &#9660;";
   button.innerHTML = "version " + chplTitle + arrow;
 
   // Choose button color
-  if (chplTitle == nextRelease) {
+  if (chplTitle == nextRelease || chplTitle == nextNextRelease) {
     button.innerHTML = "version " + chplTitle + " (pre-release)" + arrow;
```

Comment:
@bradcray @ben-albrecht 
I remembered this little detail- we are planning to take out that "pre-release" version string from line 64 of conf.py (https://github.com/chapel-lang/chapel/pull/11111/commits/fdfa83bf8fdb0afe0a15924dde30cb077ef3e470), because Brad effectively moved it to versionButton.php (above). 

But look at line 65 in conf.py, right below the former "pre-release" string (https://github.com/chapel-lang/chapel/blob/fdfa83bf8fdb0afe0a15924dde30cb077ef3e470/doc/rst/conf.py#L65)

IIRC, we put in that "non-break hyphen" to make sure the versionButton had no line break- or if it did, it happened where we wanted. 

I wonder if versionButton.php needs something like that?

Reply:
The 1.19-preview docs that I built look OK (see `~chapelu/www/1.19.0-pre/master/`, Cray internal).  Button says pre-release w/o a linebreak. At least in my browser. 

BTW, I assume we will bump the "currentRelease"/"nextRelease" numbers (line 90, versionButton.php) on Sept 20.

Reply:
Did this.  @awallace-cray, I'm planning on merging this today because it shouldn't break anything (and if it does, better to know now than later) and will set us up for a smooth transition to 1.19 tomorrow.